docs/admin/runmetrics.md

```diff
@@ -139,7 +139,7 @@ indicates the number of page faults which happened since the creation of
 the cgroup; this number can never decrease).
 
 
- - **cache:**  
+ - **cache:**
```

Comment:
oh, I restored them in https://github.com/docker/docker/issues/25118 (I assumed it wasn't changed on purpose), but we can update in a future PR; perhaps even consider using a table for it